packages/devtools_test/lib/src/mocks.dart

```diff
@@ -260,20 +261,24 @@ class FakeVmService extends Fake implements VmServiceWrapper {
     this._httpProfile,
     this._memoryData,
     this._allocationData,
-    CpuProfileData cpuProfileData,
+    CpuSamples cpuSamples,
   )   : _startingSockets = _socketProfile?.sockets ?? [],
         _startingRequests = _httpProfile?.requests ?? [],
-        cpuProfileData = cpuProfileData ??
-            CpuProfileData.parse(<String, dynamic>{
-              'type': '_CpuProfileTimeline',
+        cpuSamples = cpuSamples ??
+            CpuSamples.parse({
               'samplePeriod': 50,
+              'maxStackDepth': 12,
               'sampleCount': 0,
-              'stackDepth': 128,
+              'timeSpan': -1,
```

Comment:
why does this need to be -1?

Reply:
since it is deprecated and always returns -1 I set it to that. In hindsight I could probably leave it null though!

Reply:
leaving -1 sgtm then. Just add a quick comment explaining that

Reply:
I've removed it from the constructor since it is redundant, and only makes things more confusing :)